source/index.html.erb

```diff
@@ -1,240 +1,273 @@
 ---
 ---
 
-<div class="content-block content-block-gray-900">
-  <div class="container">
-    <div class="row align-items-center">
-      <div class="col-md">
-        <div class="block-content mb-4 mb-md-0">
-          <h1 class="title">The only eCommerce platform you’ll ever need.</h1>
-          <p class="lead">Build, customize and scale your store: no limits, no license fees. Solidus is the open-source eCommerce framework for digitally-native brands, fast-growing online businesses, and passionate developers.</p>
+<div class="dark-background">
+  <div class="homepage-hero content-block--skewed content-block-white">
+    <div class="content-wrapper container">
+      <div class="row align-items-center">
+        <div class="col-lg">
+          <div class="pr-lg-6 mb-3">
+            <h1 class="title">The only eCommerce platform you’ll ever need.</h1>
+            <p class="lead mb-6">
+              Build, customize and scale your store: no limits, no license fees. Solidus is the open-source eCommerce framework for digitally-native brands, fast-growing online businesses, and passionate developers.
+            </p>
+            <a class="btn btn-primary btn-lg" href="#">Get started</a>
```

Comment:
I would link to the Contact page, both here and in the CTA. I don't think we have anything else that would make sense right now. Maybe going forward we should have a "Get Started" page that links to GitHub, Contact Us etc.?

Reply:
I agree. We'd create a "Get Started" page that onboard new users in the future.